bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx116-fr.html

```diff
@@ -0,0 +1,91 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import picto with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_whatsnew_116_eu') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-md mzp-t-content-nospace">
+    <div class="mzp-c-video">
+      <a class="video-play js-video-play" href="https://youtu.be/_obdx3VUzuQ" data-id="_obdx3VUzuQ" data-video-title="Mozilla: The Big Picture" title="Play the video">
+        <img src="{{ static('img/firefox/whatsnew/whatsnew116-eu/video-thumbnail.jpg') }}" alt="" width="1024" height="576">
+      </a>
+    </div>
+
+    <h2 class="wnp-main-title">Regardez des vidéos en pop-up</h2>
+    <p class="wnp-main-tagline has-pip-icon">
+      Épinglez une vidéo dans la fenêtre de votre navigateur grâce à notre nouvelle fonction "Incrustation vidéo". Essayez avec la vidéo ci-dessus. Appuyez sur "lecture" et cliquez sur <strong>the Picture-in-Picture button</strong> pour faire sortir la vidéo.
```

Comment:
TODO: `<strong>the Picture-in-Picture button</strong>` still needs FR translation.